frontend/src/shared/components/StatusToolTip.tsx

```diff
@@ -1,82 +1,206 @@
-import { ITooltipParams } from "ag-grid-community";
+import { ColDef, ITooltipParams } from "ag-grid-community";
 import { AgGridReact } from "ag-grid-react";
+import { DashboardSample } from "../../generated/graphql";
 
-export const StatusTooltip = (props: ITooltipParams) => {
-  let { primaryId, validationReport, validationStatus } = props.data;
+type SampleStatusItem = {
+  status: string;
+  meaning: string;
+  actionItem: string;
+  responsibleParty: string;
+};
 
-  const columnDefs = [
-    {
-      field: "fieldName",
-      headerName: "Field Name",
-    },
+type SampleStatusMap = {
+  [key: string]: SampleStatusItem;
+};
+
+/**
+ * @Key Concatenated string of key-value pairs from validationReport field of Sample's Status in the database.
+ * Example: given "fastQs=missing,igoComplete=false", we'd get "fastQs missing" and "igoComplete false".
+ * Note: when querying this data from the database, handle both formats: JSON string and {key=value} string.
+ *
+ * @Value Actionable details for the PM team.
+ */
+const SAMPLE_STATUS_MAP: SampleStatusMap = {
+  "baitSet missing": {
+    status: "Baitset Missing",
+    meaning: "Baitset field does not have a value assigned in LIMS",
+    actionItem:
+      "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or " +
+      "clicking 'Mark delivery' button",
+    responsibleParty: "IGO",
+  },
+  "fastQs missing": {
+    status: "FASTQs Missing",
+    meaning: "FASTQs for sample were not sent to SMILE from IGO",
+    actionItem:
+      "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or " +
+      "clicking 'Mark delivery' button",
+    responsibleParty: "IGO",
+  },
+  "igoComplete false": {
+    status: "IGOComplete False",
+    meaning: "IGOComplete checkbox was not ticked in LIMS",
+    actionItem:
+      "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or " +
+      "clicking 'Mark delivery' button",
+    responsibleParty: "IGO",
+  },
+  "sample type abbreviation could not resolve based on specimenType, sampleOrigin, or sampleClass":
     {
-      field: "report",
-      headerName: "Report",
-      wrapText: true,
-      autoHeight: true,
-      flex: 2,
-      cellStyle: { wordBreak: "break-word" },
+      status: "Sample type abbreviation not resolved",
+      meaning:
+        "Metadata required for CMO Sample Name generation is missing/ incomplete",
```

Comment:
extra whitespace between "missing/" and "incomplete"? unsure if intentional so wanted to point it out